x/application/keeper/msg_server_undelegate_from_gateway_test.go

```diff
@@ -92,6 +92,9 @@ func TestMsgServer_UndelegateFromGateway_SuccessfullyUndelegate(t *testing.T) {
 	_, err = srv.UndelegateFromGateway(ctx, undelegateMsg)
```

Comment:
Coul we increase the unit and/or integration test coverage? We added A LOT of on-chain logic but zero new (non end to end tests) were added.